node.sh

```diff
@@ -0,0 +1,41 @@
+package: node
+version: "v9.8.0"
+build_requires:
+  - curl
+prefer_system: "(?!slc5)"
+prefer_system_check: |
+  node -h &> /dev/null || { echo "node is missing"; exit 1; }
+  if [[ printf "%d%04d%04d" $(node --version | sed -e 's/v//g; s/\./ /g') -le 900090000 ]]; then echo "higher node version required"; exit 1; fi
+---
+#!/bin/bash
+
+NODEOS=linux
+case $ARCHITECTURE in
+  osx*) NODEOS=darwin ;;
+esac
+
+FILE_NAME="$PKGNAME-$PKGVERSION-$NODEOS-x64"
+URL="https://nodejs.org/dist/$PKGVERSION/$FILE_NAME.tar.gz"
+cd "$INSTALLROOT"
+curl -L $URL | tar xzf -
```

Comment:
How about:

```bash
curl -L $URL | tar xzf - --strip-components=2 -C $INSTALLROOT
```

Reply:
This way you do not need any of the `cd / mv / rm`. On my mac it gives:

```
→ ls -la test-node2
total 288
drwxr-xr-x    9 ktf  staff    288 Apr 27 10:00 .
drwxr-xr-x  168 ktf  staff   5376 Apr 27 10:00 ..
-rw-r--r--    1 ktf  staff  53692 Mar  7 22:31 CHANGELOG.md
-rw-r--r--    1 ktf  staff  60375 Mar  7 22:31 LICENSE
-rw-r--r--    1 ktf  staff  27396 Mar  7 22:31 README.md
drwxr-xr-x    5 ktf  staff    160 Mar  7 22:32 bin
drwxr-xr-x    3 ktf  staff     96 Mar  7 22:31 include
drwxr-xr-x    4 ktf  staff    128 Mar  7 22:31 lib
drwxr-xr-x    5 ktf  staff    160 Mar  7 22:31 share
```

which is the final layout you want, I assume?

Reply:
`--strip-components=1` is prefect, is it okay to use it?

Reply:
Ok, it works flawlessly on both Linux and Mac